std/file.d

```diff
@@ -81,24 +81,6 @@ else version(Posix)
 }
 
 
-// @@@@ TEMPORARY - THIS SHOULD BE IN THE CORE @@@
```

Comment:
I'm guessing that because the tests pass that the imports for these definitions were already present.